java-frontend/src/main/java/org/sonar/java/se/ExplodedGraphWalker.java

```diff
@@ -993,7 +999,15 @@ public void enqueue(ExplodedGraph.ProgramPoint programPoint, ProgramState progra
     enqueue(programPoint, programState, false);
   }
 
+  public void enqueue(ExplodedGraph.ProgramPoint programPoint, ProgramState programState, @Nullable MethodYield methodYield) {
+    enqueue(programPoint, programState, false, methodYield);
+  }
+
   public void enqueue(ExplodedGraph.ProgramPoint newProgramPoint, ProgramState programState, boolean exitPath) {
+    enqueue(newProgramPoint, programState, exitPath, null);
+  }
+
+  public void enqueue(ExplodedGraph.ProgramPoint newProgramPoint, ProgramState programState, boolean exitPath, @Nullable MethodYield methodYield) {
```

Comment:
Thinking outloud (in writing) more than actually a request for change : 
I am wondering if it is a good thing to have this methodYield parameter added to this method as this is kinda corner case compared to all the other cases... not sure what would be an alternative but addition of this parameter is bothering me a bit (more a gut feeling than a real thing here)